GOVERNANCE.md

```diff
@@ -0,0 +1,95 @@
+## Overview
+
+This document outlines a set of policies in order to provide a level playing field and open process for contributors to join the OpenFGA project.
+
+Governance activities are currently performed manually. In the future we might use [terraform](https://www.terraform.io/) or similar technology to automate tasks, but we're currently a small team and iterating quickly - for now it makes sense to do things in a way that doesn't scale.
```

Comment:
How can terraform be used to automate governance activities? Isn't it more for automating deployment?

Reply:
It's just one method, there are many. With Terraform, you can pave/repave github repos, including teams and the associated permissions (name, labels / tags, members, teams, readme details, licensing, CI settings, visibility, and more can all be automated). 

![image](https://user-images.githubusercontent.com/6372810/207607050-dc47ec10-6317-4f24-b99e-81cfbde204ce.png)
You use one repo (in this diagram it's labelled 'repository manager', in our case it would more likely be called 'governance' or something similar) with configuration as code documented in a `.tfplan` file. This acts sort of like the model in OpenFGA: defining which repos have what settings, and which teams have what permissions for each repo, and so on. Then you use .yml files or something similar in the repo as your 'state' data. When someone makes a change (eg, adding themselves as a member of the `XYZ-team` with a PR that a `core` team member merges) then Terraform interprets the .yml and .tfplan and updates repos, teams, etc.